client/src/pages/Dashboard/Dashboard.tsx

```diff
@@ -20,7 +24,19 @@ export default function Dashboard(): JSX.Element {
   const [dashOptionSelected, setDashOption] = React.useState(dashOptions[0]);
   const schedOptions = ['UPCOMING', 'PENDING', 'PAST'];
   const [schedSelect, setSchedSelect] = React.useState(schedOptions[0]);
-  const meetingOptions = [15, 30, 45];
+  const loggedInUser: any = useAuth().loggedInUser;
+  const [meetingOptions, setMeetingOptions] = useState<Meetings>([]);
+
+  const fetchMeetingsCallback = useCallback(async (id) => {
+    const meetings = await fetchMeetings(id);
+    if (meetings.success) setMeetingOptions(meetings.success.data);
+  }, []);
```

Comment:
Could you explain why you choose to do `useCallback` here? Thank you! 😄

Reply:
Yeah, I used a callback here because if I called it within my useEffect I would run into infinite re-renders because setting state would trigger an infinite loop. So in order to avoid that issue, I would only call this function once I successfully create an event modal.